src/index.d.ts

```diff
@@ -81,8 +81,10 @@ interface Inputs<T, Name = keyof T> {
   checkbox(name: Name, ownValue?: OwnValue): CheckboxProps<T>;
   checkbox(options: InputOptions<T, Name, Maybe<OwnValue>>): CheckboxProps<T>;
 
-  raw<Y>(name: Name, value?: Y): RawInputProps<Y>;
-  raw<Y>(options: RawInputOptions<T, Y, Name>): RawInputProps<Y>;
+  raw<RawValue>(name: Name, ownValue?: any): RawInputProps<T, Name, RawValue>;
```

Comment:
The raw value was inferred from `ownValue` which is incorrect. `ownValue` is very unlikely with `raw` which leaves the raw value undetermined.

```diff
- raw<Y>(name: Name, value?: Y): RawInputProps<Y>;
+ raw<RawValue>(name: Name, ownValue?: any): RawInputProps<T, Name, RawValue>;
```

ownValue is usually used with inputs that have the same name but can support multiple values. It's only needed for checkboxes and radio buttons.